src/vaadin-message-input.js

```diff
@@ -65,6 +79,29 @@ class MessageInputElement extends ElementMixin(ThemableMixin(PolymerElement)) {
     // Set initial height to one row
     textarea.setAttribute('rows', 1);
     textarea.style.minHeight = '0';
+
+    // Add enter handling for text area.
+    textarea.addEventListener('keydown', (e) => {
```

Comment:
Please move all the textarea setup from here to the `ready()` method of `vaadin-message-text-area`.

Reply:
As per https://github.com/vaadin/vaadin-messages/pull/32#discussion_r577447832, we will address this in a separate PR